app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/upscan/UpscanResult.scala

```diff
@@ -15,7 +15,7 @@
  */
 
 package uk.gov.hmrc.cgtpropertydisposalsfrontend.models.upscan
-
+//TODO: Take out - not used
```

Comment:
is that something you were planning on doing in this PR or a later PR?

Reply:
have taken that out